include/libxnvme_spec.h

```diff
@@ -185,6 +187,30 @@ struct __attribute__((packed)) xnvme_spec_log_health_entry {
 };
 XNVME_STATIC_ASSERT(sizeof(struct xnvme_spec_log_health_entry) == 512, "Incorrect size")
 
+
+/**
+ * NVMe get-log-page entry for sanitize status information
+ *
+ * NVMe 1.4c - Figure 242
+ *
+ * @struct xnvme_spec_log_sanitize_status_entry
+ */
+
+struct __attribute__((packed)) xnvme_spec_log_sanitize_status_entry {
+	uint16_t sprog;         ///< Sanitize Progress 
+	uint16_t sstat;         ///< Sanitize Status : Bits 0:2 contain the status of the most recent sanitize operation
+                                ///< 000b - Never Sanitized, 001b - Operation Successful, 010b - In Progress, 011b - Failed, 100b - success 
+        uint32_t scdw10;        ///< Sanitize Command Dword 10 Information
+        uint32_t etfo;          ///< Estimated Time for Overwrite
+        uint32_t etfbe;         ///< Estimated Time for Block Erase
+        uint32_t etfce;         ///< Estimated Time for Crypto Erase
+        uint32_t etfownmm;      ///< Estimated Time for Overwrite with No-Deallocate Media Modification
+        uint32_t etfbewnmm;     ///< Estimated Time for Block Erase with No-Deallocate Media Modification
+        uint32_t etfcewnmm;     ///< Estimated Time for Crypto Erase With No-Deallocate Media Modification 
+        uint8_t rsvd[480];
```

Comment:
The newest base spec (2.1) defines acronyms for the fields etc. It would be great to comply with this.